sdk-extensions/incubator/src/main/java/io/opentelemetry/sdk/extension/incubator/fileconfig/ComposableSamplerFactory.java

```diff
@@ -43,6 +44,13 @@ public ComposableSampler create(
     if (ruleBased != null) {
       return ComposableRuleBasedSamplerFactory.getInstance().create(ruleBased, context);
     }
+    ExperimentalComposableParentThresholdSamplerModel parentThreshold = model.getParentThreshold();
+    if (parentThreshold != null) {
+      ExperimentalComposableSamplerModel rootModel =
+          FileConfigUtil.requireNonNull(parentThreshold.getRoot(), "parent threshold sampler root");
```

Comment:
doesn't look like we have tests to verify failure modes. Is it worth adding a failed sampler test to catch these cases? We could start small, just with this case, and add over time.

Reply:
Yes I'll add 👍